src/ChangeStream.php

```diff
@@ -39,8 +39,10 @@ class ChangeStream implements Iterator
     private $resumeCallable;
     private $csIt;
     private $key;
+    private $hasAdvanced;
 
-    private $previousValid;
+    $this->key = 0;
+    $this->hasAdvanced = false;
```

Comment:
Declarations should be:

```php
private $key = 0;
private $hasAdvanced = false;
```

I assume this was a typo copying content from the constructor, but if not: http://php.net/manual/en/language.oop5.properties.php#example-180